datarobot_batch_scoring/utils.py

```diff
@@ -433,6 +435,37 @@ def get_rusage():
         return {}
 
 
+DialectTuple = namedtuple('DialectTuple',
+                          'delimiter doublequote escapechar lineterminator '
+                          'quotechar quoting skipinitialspace strict')
+
+
+class SerializableDialect(DialectTuple):
+    @classmethod
+    def from_dialect(cls, dialect):
+        return cls(
+            delimiter=dialect.delimiter,
+            doublequote=dialect.doublequote,
+            escapechar=dialect.escapechar,
+            lineterminator=dialect.lineterminator,
+            quotechar=dialect.quotechar,
+            quoting=dialect.quoting,
+            skipinitialspace=dialect.skipinitialspace,
+            strict=getattr(dialect, 'strict', False))
+
+    def to_dialect(self):
```

Comment:
it looks hackish... Why you don't just create csv.Dialect instance passing all parameters to constructor?